app/lang/en.json

```diff
@@ -288,31 +288,30 @@
     "readYourLpa": "Read and sign your LPA",
     "readAndCheckYourLpa": "Read and check your LPA carefully before you sign it.",
     "whatHappensIfIWantToMakeChanges": "What happens if I want to make changes after I’ve signed?",
-    "whatHappensIfIWantToMakeChangesContent": "<p class=\"govuk-body\">It is possible to change your LPA after you’ve signed it and had the conversation with your certificate provider. However, you will need to:</p> <ul class=\"govuk-list govuk-list--bullet\"> <li>sign the LPA again</li> <li>have another conversation with your certificate provider, so you can talk about the updated LPA.</li> </ul> <p class=\"govuk-body\">Once the LPA has been registered, you can remove an attorney, but cannot make any other changes.</p><p> If you want to make other changes, you can cancel the LPA and make a new one. </p>",
+    "whatHappensIfIWantToMakeChangesContent": "<p class=\"govuk-body\">Guidance</p>",
     "iHaveReadAndCheckedThisLpa": "I have read and checked this LPA",
     "iConfirmIMadeThisLpaOfMyOwnFreeWill": "I confirm that I made this LPA of my own free will",
     "typeSignatureCode": "Type your signature code to sign",
     "iHaventReceivedSignatureCode": "I haven’t received a signature code",
-    "continueToSigningPage": "Continue to signing page",
 
     "signYourLpa": "Sign your LPA",
     "signYourLpaHeader": "<h2 class=\"govuk-heading-m\">Signing the LPA is an important and powerful legal step.</h2>",
-    "wantAttorneysToBeAppointed": {
-        "one": "When you do this you’re officially saying that you want {{ .AllAttorneysFullNames }} to be your {{ .LpaType }} attorney.",
-        "other": "When you do this you’re officially saying that you want {{ .AllAttorneysFullNames }} to be your {{ .LpaType }} attorneys."
+    "bySigningIConfirm": "By signing my {{ .TypeLegalTermTransKey}} LPA, I confirm all of the following:",
```

Comment:
can we keep the placeholder values as "friendly" `LpaType` since we are in full control of the names